src/objects/BatchedMesh.js

```diff
@@ -144,6 +144,7 @@ class BatchedMesh extends Mesh {
 
 		// stores visible, active, and geometry id per object
 		this._drawInfo = [];
+		this._fragInfo = [];
```

Comment:
What does the name "_fragInfo" refer to? It would be better to call this something like "availableInstanceIds"

Reply:
Sounds good, I like the name "availableInstanceIds". Will make that change